tailwind.config.js

```diff
@@ -18,6 +18,13 @@ module.exports = {
         '3xl': '1920px',
         '4xl': '2560px',
       },
+      transitionDuration: {
+        300: '300ms',
+      },
+      translate: {
+        0: '0',
+        '-full': '-100%',
+      },
```

Comment:
Where is this being used? I think we can achieve through pure tailwind classmame @micohalvarez